controllers/admin/config.go

```diff
@@ -416,7 +416,8 @@ func SetServerURL(w http.ResponseWriter, r *http.Request) {
 	}
 
 	rawValue, ok := configValue.Value.(string)
-	if !ok {
+	serverHostString := utils.GetHostnameFromURLString(rawValue)
```

Comment:
If `ok` is false then `rawValue` I'm assuming is an empty string. I'm not sure what happens when an empty string is passed to `GetHostnameFromURLString`.

Reply:
it calls net/url.Parse() which returns empty string and no error

Reply:
I just separated the checks just in case.